tests/unit/Jadu/Pulsar/Twig/Macro/Fixtures/form/toggle_switch-error.html

```diff
@@ -1,4 +1,4 @@
-<div class="form__group has-error">
+<div class="form__group form__group--toggle has-error">
```

Comment:
Because of the order that classes are being merged in the `form.html.twig` helper, they'll be rendered as  `form__group has-error form__group--toggle`, so this test will currently fail.